### PR TITLE
add logfile, symlink to manifest;

### PR DESCRIPTION
**Pull request by lduabc**
get logfile from wisktrack and save to cache storage, print it out when copying from cache;
get symlink from wisktrack and create symlinks when cache out;
flatten the structure of cache output storage.

### CODE REVIEW

File: cache/cache.go [hunk 1]
```diff
@@ -89,15 +90,16 @@ func Create(config config.Config, inFile []string, outFile []string, manifestfil
...
-            }
+        target := filepath.Join(dirOfCachedOutputFiles, strings.Replace(ofile, "/", ".", -1))
+        cpCmd := exec.Command("cp", filepath.Join(config.BaseDir, ofile), target)
+        err = cpCmd.Run()
+        if err != nil{
+            return err
         }
-        cpCmd := exec.Command("cp", filepath.Join(config.BaseDir, ofile),
-                              filepath.Join(fullPath, filepath.Base(ofile)))
+    }
```
Comment (sarvi) [inline cache/cache.go:99]: We might want to use goroutines to parallelize the copy/symlink operations. This commit is fine. But in a subsequent commit, we should do that.

File: cache/cache.go [hunk 2]
```diff
@@ -133,8 +135,26 @@ func CopyOut(config config.Config, manifestFile string)(error){
...
+            if err != nil{
+                fmt.Printf("Failed to open %v\n", logFile)
+            }else{
+                scanner := bufio.NewScanner(file)
+                scanner.Split(bufio.ScanLines)
+                for scanner.Scan() {
+                    fmt.Println(scanner.Text())
+                }
+            }
+        }
```
Comment (sarvi) [inline cache/cache.go:155]: Do we have to read and print line by line. something like this https://github.com/tokuhirom/golang-samples/blob/master/samples/coreutils/cat.go ?

**Review by sarvi — APPROVED** [review]

File: cache/cache.go (hunk 1 above)
Comment (lduabc) [reply cache/cache.go]: took care of them in the new commit

File: cache/cache.go (hunk 2 above)
Comment (lduabc) [reply cache/cache.go:183]: took care of it in the new push

**Pull request closed without merging**